tests/suites/test_suite_ecp.data

```diff
@@ -336,6 +336,14 @@ ECP test vectors Curve448 (RFC 7748 6.2, after decodeUCoordinate)
 depends_on:MBEDTLS_ECP_DP_CURVE448_ENABLED
 ecp_test_vec_x:MBEDTLS_ECP_DP_CURVE448:"eb7298a5c0d8c29a1dab27f1a6826300917389449741a974f5bac9d98dc298d46555bce8bae89eeed400584bb046cf75579f51d125498f98":"a01fc432e5807f17530d1288da125b0cd453d941726436c8bbd9c5222c3da7fa639ce03db8d23b274a0721a1aed5227de6e3b731ccf7089b":"ad997351b6106f36b0d1091b929c4c37213e0d2b97e85ebb20c127691d0dad8f1d8175b0723745e639a3cb7044290b99e0e2a0c27a6a301c":"0936f37bc6c1bd07ae3dec7ab5dc06a73ca13242fb343efc72b9d82730b445f3d4b0bd077162a46dcfec6f9b590bfcbcf520cdb029a8b73e":"9d874a5137509a449ad5853040241c5236395435c36424fd560b0cb62b281d285275a740ce32a22dd1740f4aa9161cec95ccc61a18f4ff07"
 
+ECP point multiplication rng fail secp256r1
```

Comment:
Do we have rng fail test for tiny ecc ?

Reply:
Our random implementation can't fail. 
They have a mechanism for internal rng to fail in ECP here: https://github.com/PelionIoT/pelion-crypto/pull/19/files/fef1be1e86878ae15bf28641f59e3f27a457419e#diff-bc5fbd4345d02975f5698ac4ba2b16c767937a870392322aa3c66c3df58a8ee4R367